buildah/README.md

```diff
@@ -21,6 +21,8 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/build
   note below.  (_default:_ quay.io/buildah/stable:v1.11.0)
 * **DOCKERFILE**: The path to the `Dockerfile` to execute (_default:_
   `./Dockerfile`)
+* **CONTEXT**: Path to the directory to use as context (_default:_
```

Comment:
Using the name `CONTEXT` brings parity with other builder tools like kaniko, easy to swap tasks if they have the same input names.
Ref: https://github.com/tektoncd/catalog/blob/master/kaniko/kaniko.yaml#L11